scripts/copy_test_gvcfs.py

```diff
@@ -0,0 +1,11 @@
+"""Copies a subset of gVCFs from the main bucket to the test bucket."""
```

Comment:
Nitpick: did we settle on gVCF or GVCF?

Reply:
We didn't -- the GATK documentation itself is totally inconsistent. See e.g. https://gatk.broadinstitute.org/hc/en-us/articles/360036827351-CombineGVCFs :)